src/containerapp/azext_containerapp/commands.py

```diff
@@ -222,4 +222,9 @@ def load_command_table(self, args):
         g.custom_show_command('list', 'list_session_pool')
         g.custom_command('create', 'create_session_pool', supports_no_wait=True)
         g.custom_command('update', 'update_session_pool', supports_no_wait=True)
-        g.custom_command('delete', 'delete_session_pool', confirmation=True, supports_no_wait=True)
\ No newline at end of file
+        g.custom_command('delete', 'delete_session_pool', confirmation=True, supports_no_wait=True)
+
+    with self.command_group('containerapp java logger') as g:
```

Comment:
Recommend add `is_preview=True` in the command group directly.
```suggestion
    with self.command_group('containerapp java logger', is_preview=True) as g:
```

Reply:
The difference:
![image](https://github.com/Azure/azure-cli-extensions/assets/46143499/85bac795-e80b-47bf-bf44-7b4b85fa8436)

![image](https://github.com/Azure/azure-cli-extensions/assets/46143499/f1ef0428-3e38-42b0-a1c7-97b874d2bb78)